build.sbt

```diff
@@ -3824,26 +3827,8 @@ lazy val `engine-runner` = project
       core ++ stdLibsJars ++ extraNITestLibs.value
     },
     extraNITestLibs := Def.taskDyn {
-      if (GraalVM.EnsoLauncher.test) Def.task {
```

Comment:
- those mysterious failures in #14019 
- could be caused by the _dual JVM mode_ used when testing
- that could be verified by reverting back these lines
- then the test helpers would be included again in `enso` executable when `ENSO_LAUNCHER=test` mode is on (hopefully)